packages/testing/config/storybook/main.js

```diff
@@ -10,80 +10,118 @@ const {
   importStatementPath,
 } = require('@redwoodjs/project-config')
 
-const config = getConfig()
-
-const rwjsPaths = getPaths()
-
-const staticAssetsFolder = path.join(getPaths().web.base, 'public')
+const redwoodProjectConfig = getConfig()
+const redwoodProjectPaths = getPaths()
 
+// Storybook docs on this file: https://storybook.js.org/docs/react/configure/overview.
+/** @type { import('@storybook/react-webpack5').StorybookConfig } */
 const baseConfig = {
-  core: {
-    builder: 'webpack5',
+  framework: {
+    name: '@storybook/react-webpack5',
+    // This empty object is actually necessary.
+    options: {},
   },
+
+  // Storybook uses picomatch for globbing; see https://github.com/micromatch/picomatch#globbing-features.
   stories: [
     `${importStatementPath(
-      rwjsPaths.web.src
+      redwoodProjectPaths.web.src
     )}/**/*.stories.@(js|jsx|ts|tsx|mdx)`,
   ],
+
+  // See https://storybook.js.org/docs/react/configure/storybook-addons.
   addons: [
     '@storybook/addon-essentials',
-    config.web.a11y && '@storybook/addon-a11y',
+    redwoodProjectConfig.web.a11y && '@storybook/addon-a11y',
   ].filter(Boolean),
-  // Storybook's UI uses a separate Webpack configuration
-  managerWebpack: (sbConfig) => {
-    const userManagerPath = fs.existsSync(rwjsPaths.web.storybookManagerConfig)
-      ? rwjsPaths.web.storybookManagerConfig
-      : './manager.example.js'
-    sbConfig.resolve.alias['~__REDWOOD__USER_STORYBOOK_MANAGER_CONFIG'] =
-      userManagerPath
 
-    return sbConfig
-  },
```

Comment:
The ability to configure the manager is gone and is the most obvious breaking change